### PR TITLE
ADD Environment variable configuration

### PR DESCRIPTION
**Pull request by dmoranj**
Implements #75

### CODE REVIEW

**Comment by AlvaroVega** [comment]
LGTM

File: lib/bindings/MQTTBinding.js [hunk 1]
```diff
@@ -311,7 +311,7 @@ function start(callback) {
     mqttClient = mqtt.connect('mqtt://' + config.getConfig().mqtt.host + ':' + config.getConfig().mqtt.port, options);
     mqttClient.on('message', mqttMessageHandler);
 
-    mqttClient.on('connect', function() {
+    mqttClient.on('connect', function(error) {
```
Comment (gtorodelvalle) [inline lib/bindings/MQTTBinding.js]: This `error` argument is not used or needed :)

File: test/unit/startup-test.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            delete process.env.MQTT_PASSWORD;
+            delete process.env.HTTP_HOST;
+            delete process.env.HTTP_PORT;
+        });
+
+        afterEach(function(done) {
+            iotagentUl.stop(done);
+        });
+
+        it('should not start and raise a MISSING_CONFIG_PARAMS error', function(done) {
```
Comment (gtorodelvalle) [inline test/unit/startup-test.js]: Is this test text right? :)

Comment (dmoranj) [inline test/unit/startup-test.js]: Not at all :D Fixed in 	b469bd4

**Comment by gtorodelvalle** [comment]
LGTM

**Pull request merged**